build.py

```diff
@@ -591,50 +592,29 @@ def main():
         apply_colloid_patches()
 
     palette = getattr(PALETTE, args.flavor)
-    accents = [
-        "rosewater",
-        "flamingo",
-        "pink",
-        "mauve",
-        "red",
-        "maroon",
-        "peach",
-        "yellow",
-        "green",
-        "teal",
-        "sky",
-        "sapphire",
-        "blue",
-        "lavender",
-    ]
-
-    if args.accent == "all":
-        for accent in accents:
-            accent = getattr(palette.colors, accent)
-
-            tweaks = Tweaks(tweaks=args.tweaks)
-
-            if args.zip:
-                output_format = "zip"
-            else:
-                output_format = "dir"
-
-            ctx = BuildContext(
-                build_root=args.dest,
-                theme_name=args.name,
-                flavor=palette,
-                accent=accent,
-                size=args.size,
-                tweaks=tweaks,
-                output_format=output_format,
-            )
 
-            tweaks_temp()
-            gnome_shell_version()
-            build_theme(ctx)
-            logger.info("Done!")
-    else:
-        accent = getattr(palette.colors, args.accent)
+    accents = args.accents
+    if "all" in accents:
```

Comment:
Hmm. Not sure how much I like the special value being kept around. Is there a better way to do this? Perhaps a flag like `--all-accents`?